core/dbt/task/compile.py

```diff
@@ -104,6 +104,12 @@ def _runtime_initialize(self):
                 )
                 sql_node = block_parser.parse_remote(self.args.inline, "inline_query")
                 process_node(self.config, self.manifest, sql_node)
+                # Special hack to remove disabled, if it's there. This would only happen
+                # if all models are disabled in dbt_project
```

Comment:
So if _all_ models are disabled, we effectively re-enable all of them? I think I'm missing something here as to why this would be safe to do

Reply:
The only way that an inline node would be disabled is if somebody has all models to enabled: false (like in the test case) because there's no place else for enabled: false to come from. This just enables and moves the temporary inline node that was just created. It doesn't affect anything else in the manifest.